include/private/gc_priv.h

```diff
@@ -857,7 +857,9 @@ EXTERN_C_END
 # endif
 #endif /* DARWIN */
 
-#include <setjmp.h>
+#ifndef GC_NO_SIGSETJMP
```

Comment:
No, this is not correct - setjmp.h is included to have setjmp() which is used if no sigsetjmp(). It seems we need ifndef WASI